packages/sdk/generated/auth/client.ts

```diff
@@ -281,6 +281,21 @@ export class AuthClient {
     return this.createDelegatedRegistrationChallenge(request)
   }
 
+  async createDelegatedRegistrationChallengeWithSocialLoginProviders(request: T.CreateDelegatedRegistrationChallengeWithSocialLoginProvidersRequest): Promise<T.CreateDelegatedRegistrationChallengeWithSocialLoginProvidersResponse> {
```

Comment:
That's quite the type parameter 😅

Reply:
yeah I thought I renamed it before merging my PR but it seems not, I will probably change that :D